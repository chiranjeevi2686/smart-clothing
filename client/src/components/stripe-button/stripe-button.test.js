import React from 'react';
import { render } from '@testing-library/react';
import StripeCheckoutButton from './stripe-button.component';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';

const defaultStore = {
    price: 10
};
const mockStore = configureStore();
describe('StripeCheckoutButton component', () => {
    it('should render StripeCheckoutButton', () => {
        const { asFragment } = render(<Provider store={mockStore(defaultStore)}>
            <StripeCheckoutButton />
        </Provider>)
        expect(asFragment()).toMatchSnapshot();
    })
})

