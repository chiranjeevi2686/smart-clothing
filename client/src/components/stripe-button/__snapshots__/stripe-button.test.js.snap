// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StripeCheckoutButton component should render StripeCheckoutButton 1`] = `
<DocumentFragment>
  <button
    class="StripeCheckout"
    style="overflow: hidden; display: inline-block; border: 0px; padding: 1px; text-decoration: none; border-radius: 5px; box-shadow: 0 1px 0 rgba(0,0,0,0.2); cursor: pointer; visibility: visible; user-select: none;"
  >
    <span
      style="font-family: \\"Helvetica Neue\\",Helvetica,Arial,sans-serif; font-size: 14px; position: relative; padding: 0px 12px; display: block; height: 30px; line-height: 30px; color: rgb(255, 255, 255); font-weight: bold; box-shadow: inset 0 1px 0 rgba(255,255,255,0.25); text-shadow: 0 -1px 0 rgba(0,0,0,0.25); border-radius: 4px;"
    >
      Pay Now
    </span>
  </button>
</DocumentFragment>
`;
